PDF/Viewer/IPDFViewer.SuperMemo.cs

```diff
@@ -225,7 +225,7 @@ protected ContentBase CreateImageContent(Image  image, string title)
 
     // PDF Extracts
 
-    protected bool CreatePDFExtract()
+    protected bool CreatePDFExtract(double priority = -1)
```

Comment:
Why is `priority` never used ?

Reply:
Sorry, I think I added that by mistake. I removed it in the latest commit.